content/operate/kubernetes/upgrade/upgrade-olm.md

```diff
@@ -85,12 +86,70 @@ oc apply -f openshift/scc.yaml
+
+1. Edit the REC custom resource YAML file.
+
+    ```sh
+    oc edit rec <your-rec.yaml>
+    ```
+
+1. Replace the `versionTag:` declaration under `redisEnterpriseImageSpec` with the new version tag.
+
+    ```YAML
+    spec:
+      redisEnterpriseImageSpec:
+        imagePullPolicy:  IfNotPresent
+        repository:       redislabs/redis
+        versionTag:       <new-version-tag>
+    ```
+
+1. Save the changes to apply.
+
+### Reapply roles and role bindings
+
+If your operator is monitoring multiple namespaces, you'll need to [reapply your role and role bindings]({{< relref "/operate/kubernetes/re-clusters/multi-namespace#create-role-and-role-binding-for-managed-namespaces" >}}) for each managed namespace. See [Manage databases in multiple namespaces]({{< relref "/operate/kubernetes/re-clusters/multi-namespace" >}}) for more details.
+
+### Monitor the upgrade
+
+You can view the state of the REC with `oc get rec`.
+
+  During the upgrade, the state should be `Upgrade`.
+  When the upgrade is complete and the cluster is ready to use, the state will change to `Running`.
+  If the state is `InvalidUpgrade`, there is an error (usually relating to configuration) in the upgrade.
+
+```sh
+$ oc get rec
+NAME   NODES   VERSION      STATE     SPEC STATUS   LICENSE STATE   SHARDS LIMIT   LICENSE EXPIRATION DATE   AGE
+rec    3       6.2.10-107   Upgrade   Valid         Valid           4              2022-07-16T13:59:00Z      92m
+```
+
+To see the status of the current rolling upgrade, run:
+
+```sh
+oc rollout status sts <REC_name>
+```
 
-### Upgrade databases
+## Upgrade databases
 
-After the cluster is upgraded, you can upgrade your databases. Specify your new database version in the `spec.redisVersion` field for your REDB and REAADB custom resources. Supported database versions for operator version 7.8.2-6 include `"7.2"` and `"7.4"` (note this value is a string).
+After the cluster is upgraded, you can upgrade your databases. Specify your new database version in the `spec.redisVersion` field for your REDB and REAADB custom resources. Supported database versions for this operator version include `"7.2"` and `"7.4"` (note this value is a string).
 
 Note that if your cluster [`redisUpgradePolicy`]({{<relref "/operate/kubernetes/reference/redis_enterprise_cluster_api#redisupgradepolicy" >}}) or your database [`redisVersion`]({{< relref "/operate/kubernetes/reference/redis_enterprise_database_api#redisversion" >}}) are set to `major`, you won't be able to upgrade those databases to minor versions. See [Redis upgrade policy]({{< relref "/operate/rs/installing-upgrading/upgrading#redis-upgrade-policy" >}}) for more details.
+The Redis Enterprise cluster (REC) can be updated automatically or manually. To trigger automatic upgrade of the REC after the operator upgrade completes, specify `autoUpgradeRedisEnterprise: true` in your REC spec. If you don't have automatic upgrade enabled, follow the below steps for the manual upgrade.
```

Comment:
I guess this should be removed? Already mentioned under the `Upgrade the Redis Enterprise Cluster` section, which seems the more relevant place.